hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/metrics/NameNodeMetrics.java

```diff
@@ -87,6 +87,8 @@
   MutableGaugeInt blockOpsQueued;
   @Metric("Number of blockReports and blockReceivedAndDeleted batch processed")
   MutableCounterLong blockOpsBatched;
+  @Metric("Number of edit pending")
+  MutableGaugeInt editPendingCount;
```

Comment:
I'm not a native English speaker, but I feel the following is more natural. What do you think?
```suggestion
  @Metric("Number of pending edits")
  MutableGaugeInt pendingEditsCount;
```

Reply:
Could you add the description to `./hadoop-common-project/hadoop-common/src/site/markdown/Metrics.md`?

Reply:
> Could you add the description to `./hadoop-common-project/hadoop-common/src/site/markdown/Metrics.md`?

Thanks for your review. I think your suggestion is reasonable. I fixed it.